<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Can J Gastroenterol Hepatol</span><span class="iso-abbrev" title="iso-abbrev">Can J Gastroenterol Hepatol</span><span class="publisher-id" title="publisher-id">CJGH</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Canadian Journal of Gastroenterology &amp;amp; Hepatology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2291-2789</span><span class="issn-epub" tagx="issn" title="issn-epub">2291-2797</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Hindawi</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7378597">7378597</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1155/2020/9701920">10.1155/2020/9701920</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Transmission of HIV and HCV within Former Soviet Union Countries</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aibekova</span><span class="given-names" tagx="given-names" title="given-names">Lazzat</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bexeitova</span><span class="given-names" tagx="given-names" title="given-names">Aizada</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aldabergenova</span><span class="given-names" tagx="given-names" title="given-names">Arailym</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hortelano</span><span class="given-names" tagx="given-names" title="given-names">Gonzalo</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Zhangwen</span></span><a href="#I2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">Feng</span></span><a href="#I2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">Yiming</span></span><a href="#I2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeHovitz</span><span class="given-names" tagx="given-names" title="given-names">Jack</span></span><a href="#I3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vermund</span><span class="given-names" tagx="given-names" title="given-names">Sten H.</span></span><a href="#I4"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-9821-1513</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">Syed</span></span><span class="email" tagx="email" title="email">syed.ali@nu.edu.kz</span><a href="#I5"><sup>5</sup></a></span></div><span class="citation_author_institution" id="I1">[], <sup>1</sup></span><span class="citation_author_institution" id="I2">[], <sup>2</sup></span><span class="citation_author_institution" id="I3">[], <sup>3</sup></span><span class="citation_author_institution" id="I4">[], <sup>4</sup></span><span class="citation_author_institution" id="I5">[], <sup>5</sup></span><div class="author-notes" title="author-notes"> <div class="fn-type-other" title="other"> <p>Academic Editor: Kevork M. Peltekian</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-7-7</span></span><span class="volume" tagx="volume" title="volume">2020</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">9701920</span><span class="history" title="history"><span class="received" title="received">received: 2020-1-17</span><span class="accepted" title="accepted">accepted: 2020-4-30</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p> Following the collapse of the Union of Soviet Socialist Republic (USSR) in 1991, trans-border mobility increased within the former Soviet Union (FSU) countries. In addition, drug-trafficking and injection drug use began to rise, leading to the propagation and transmission of blood-borne infections within and across the FSU countries. To examine the transmission of blood-borne infections within this region, we analyzed the phylogenetic relationship of publically available sequences of two blood-borne viruses, hepatitis C virus (HCV) and human immunodeficiency virus (HIV), from FSU countries. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p> We analysed 614 and 295 NS5B sequences from HCV genotypes 1b and 3a, respectively, from 9 FSU countries. From 13 FSU countries, we analysed 347 HIV <i>gag</i> and 1282 HIV <i>env</i> sequences. To examine transmission networks and the origins of infection, respectively, phylogenetic and Bayesian analyses were performed. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p> Our analysis shows intermixing of HCV and HIV sequences, suggesting transmission of these viruses both within and across FSU countries. We show involvement of three major populations in transmission: injection drug user, heterosexual, and trans-border migrants. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p> This study highlights the need to focus harm reduction efforts toward controlling transmission of blood-borne infections among the abovementioned high-risk populations in the FSU countries.</p> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Nazarbayev University</div><span class="award-id" tagx="award-id" title="award-id">110119FD4516</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National Institutes of Health</div><span class="award-id" tagx="award-id" title="award-id">P30MH062294</span><span class="award-id" tagx="award-id" title="award-id">D43 TW010046</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Fogarty International Center</div> </div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Following the collapse of the Union of Soviet Socialist Republics (USSR) in 1991, the ensuing economic crisis led to poverty and unemployment in the former Soviet republics. Existing social and cultural ties among the former Soviet Union (FSU) countries and visa-free travel across borders facilitated massive movement of migrants in search of employment [<a href="#B1">1</a>]. High migration rates in the setting of economic destabilization were accompanied by increased rates of injected drug use, facilitating the transmission of blood-borne viruses such as human immunodeficiency virus (HIV) and hepatitis C virus (HCV) in the region [<a href="#B2">2</a>, <a href="#B3">3</a>]. Due to their geographic location along drug-trafficking routes from Afghanistan, the main hub of opium production and supply for Russia and Europe, there has been increased trafficking and use of injectable drugs in Central Asia [<a href="#B4">4</a>]. According to the World Health Organization (WHO), in Eastern Europe, 6.8 million people were estimated in 2015 to be positive for antibodies to HCV (3.3% prevalence) and 4.7 million people were living with chronic HCV (2.3% prevalence; 69% viremia rate), while in Central Asia, these figures were 4.5 million (5.4% prevalence) and 1.9 million (2.3% prevalence; 43% viremia rate) people [<a href="#B5">5</a>]. The number of people living with HIV in Eastern European and Central Asian (EECA) countries, representing the only region in the world with rising HIV incidence, reached 1.6 million by 2016 [<a href="#B6">6</a>].</p> <p>The objective of this study was to investigate the epidemiology of blood-borne viruses, namely, HIV and HCV, among FSU countries. Using viral sequences from public databases, we have performed phylogenetic analysis to assess common routes of transmission of these two viruses within FSU countries.</p> </div> <div class="2.methods" title="sec"> <div class="title" tagx="title" title="title">
2. Methods</div> <div class="2.1.selectionanddownloadingofsequences" title="sec"> <div class="title" tagx="title" title="title">
2.1. Selection and Downloading of Sequences</div> <p>For HCV, a 234â€‰bp fragment of NS5B gene (corresponding to H77 8322â€"8555â€‰nt) was studied. We downloaded 614 sequences for genotype 1b and 295 for genotype 3a from the Los Alamos HCV database (<a href="http://www.hcv.lanl.gov">http://www.hcv.lanl.gov</a>). Sequences from 9 FSU countries, namely, Russia, Uzbekistan, Tajikistan, Azerbaijan, Belarus, Lithuania, Latvia, Estonia, and Georgia, were used for this analysis. Sequence retrieval and length decisions were based on selecting the most represented gene fragment (and genotype sequences) available for most FSU countries in the said database (Supplementary <a href="#supplementary-material-1">Figure 1</a>). Genotypes for all the sequences were ascertained using the Oxford HCV Automated Subtyping Tool 2.0 (<a href="http://www.bioafrica.net/rega-genotype/html/subtypinghcv.html">http://www.bioafrica.net/rega-genotype/html/subtypinghcv.html</a>). For constructing phylogenetic tress, 10 known genotype sequences were used as reference.</p> <p>For HIV, subtype A <i>gag</i> and <i>env</i> sequences from Los Alamos HIV Sequence Database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov</a>) were downloaded. Available sequences from 13 FSU countries, namely, Armenia, Azerbaijan, Belarus, Estonia, Georgia, Kazakhstan, Kyrgyzstan, Latvia, Lithuania, Moldova, Russia, Ukraine, and Uzbekistan, were retrieved (Supplementary <a href="#supplementary-material-1">Figure 1</a>). Recombinant and duplicate sequences were identified using, respectively, <i>Recombinant HIV-1 Drawing Tool v2.1.0</i> and <i>ElimDupes</i> (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov</a>), and eliminated. After alignment and trimming, 371â€‰bp long, 334 <i>gag</i> sequences (HXB2: 859â€"1230â€‰nt), and 257â€‰bp long, 1282 <i>env</i> sequences (HXB2: 7071â€"7336â€‰nt) were used for this analysis. To root the tree, 12 reference sequences of nonsubtype A and group M were used from Los Alamos HIV Sequence Database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov</a>). To construct the tree with risk group information, data from the HIV Los Alamos database were used.</p> </div> <div class="2.2.phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.2. Phylogenetic Analysis</div> <p><i>SeaView v4.6.1</i> was used to align sequences using the ClustalO algorithm, and the alignments were manually edited using <i>MEGA v6.06</i>. The phylogenetic trees were constructed using maximum likelihood method with 100 replicates bootstrap evaluation under the Tamura-Nei model as nucleotide substitution model, including a uniform rate of heterogeneity among sites, as implemented in <i>MEGAv6.06.</i> The final tree was refined and visualized using Figtree 1.4.2 software (<a href="http://tree.bio.ed.ac.uk/software/">http://tree.bio.ed.ac.uk/software/</a>). The clusters were picked based on the predominance of sequences from a particular country or risk group.</p> </div> <div class="2.3.bayesiananalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3. Bayesian Analysis</div> <p>To investigate temporal signal in the data set, we used the maximum-likelihood tree for the analysis of correlation between root-to-tip genetic distance and year of sampling in the program <i>TempEst v1.5.</i> A Bayesian phylogenetic approach [<a href="#B7">7</a>] was used for joint estimation of the ages of each of clusters and the demographic history of all of the strains. This was done by analyzing these sequences using the general time reversible (GTR) model [<a href="#B8">8</a>] plus a gamma distribution among site rate heterogeneities [<a href="#B9">9</a>] on the basis of the standard Akaike information criterion in a hierarchical likelihood ratio test in our study. Bayesian Markov chain Monte Carlo (MCMC) analyses were performed with the selected nucleotide substitution model using an uncorrelated lognormal relaxed-clock model [<a href="#B10">10</a>] as implemented in <i>BEAST, version 1.8.4.</i> The analyses were performed using a Bayesian SkyGrid coalescent tree prior [<a href="#B11">11</a>, <a href="#B12">12</a>] in order to investigate the demographic histories of HCV or HIV and the degree to which dating estimates are affected by the demographic model chosen. Sequences for which sampling year information was unavailable were excluded from the analysis.</p> <p>To ensure convergence of parameter estimates, the MCMC was run for 100 million steps and sampled every 10000 steps. The resulting MCMC samples under different demographic models were used to estimate the rates of evolution and tMRCAs with the first 10% removed as burn-in. Convergence of parameters and sufficient sampling were assessed by calculating the effective sample size (ESS) using TRACER v1.6 (<a href="http://beast.bio.ed.ac.uk/software/tracer">http://beast.bio.ed.ac.uk/software/tracer</a>). A final maximum clade credibility (MCC) tree was constructed from the posterior tree distribution using the program Tree Annotator (<a href="http://beast.bio.ed.ac.uk/">http://beast.bio.ed.ac.uk</a>) and visualized in <i>FigTree.</i></p> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.hcv" title="sec"> <div class="title" tagx="title" title="title">
3.1. HCV</div> <p>The maximum likelihood trees generated using genotype 1b or 3a NS5B sequences from 7 countries revealed phylogenetic links of Russian clusters with sequences from other FSU countries. The sequences analysed were deposited during 1995â€"2015 (Tables <a href="#tab1">1</a> and <a href="#tab2">2</a>). Overall, 10 clusters were picked from HCV 1b NS5B tree based on the predominance of sequences from a particular country or risk group (<a href="#tab1">Table 1</a> and <a href="#fig1">Figure 1</a>): cluster A (53 sequences, representing 5 countries), cluster B (37 sequences, representing 7 countries), cluster C (18 sequences, representing 4 countries), cluster D (23 sequences, representing 6 countries), cluster E (133 sequences, representing 6 countries), cluster F (33 sequences, representing 6 countries), cluster G (30 sequences, representing 5 countries), cluster H (31 sequences, representing 4 countries), cluster I (39 sequences, representing 6 countries), and cluster J (32 sequences, representing 5 countries). Genotype 1b tree showed intermixing of Russian sequences mainly with those from Estonia (clusters A, B, and Dâ€"J), Lithuania (clusters B-J), Azerbaijan (clusters A-E, G, and I), Uzbekistan (clusters A-G and I), and Tajikistan (clusters A, B, Dâ€"F, and Hâ€"J) (<a href="#fig1">Figure 1</a>, and <a href="#tab1">Table 1</a>). Seven clusters were picked from HCV 3a NS5B tree based on the predominance of sequences from a particular country or risk group (<a href="#tab2">Table 2</a> and <a href="#fig2">Figure 2</a>): cluster A (44 sequences, representing 6 countries), cluster B (11 sequences, representing 3 countries), cluster C (11 sequences, representing 4 countries), cluster D (18 sequences, representing 3 countries), cluster E (23 sequences, representing 3 countries), cluster F (80 sequences, representing 6 countries), and cluster G (33 sequences, representing 4 countries). Genotype 3a tree showed a similar trend with an intermixing of HCV sequences from Russia mainly with those from Estonia (clusters A and Câ€"G), Uzbekistan (clusters A, C, and Eâ€"G), and Azerbaijan (clusters A, B, D, F, and G) (<a href="#fig2">Figure 2</a> and <a href="#tab2">Table 2</a>).</p> </div> <div class="3.2.hiv" title="sec"> <div class="title" tagx="title" title="title">
3.2. HIV</div> <p>A phylogenetic tree was constructed with <i>env</i> sequences from 10 counties, spanning years 1993â€"2015, as follows: Armenia (2009), Belarus (1997â€"2015), Georgia (1997â€"2013), Estonia (2001â€"2005), Kyrgyzstan (2009), Latvia (1998â€"2005), Moldova (1997), Russia (1994â€"2015), Ukraine (1993â€"2012), and Uzbekistan (1999â€"2002). In addition, 12 nonsubtype A references were retrieved from the database and were used as outliers to root the phylogenetic tree. Fifteen clusters were picked from the tree for further analysis (<a href="#tab3">Table 3</a> and <a href="#fig3">Figure 3</a>): cluster A (76 sequences, representing 5 countries), cluster B (53 sequences, representing 4 countries), cluster C (25 sequences, representing 2 countries), cluster D (83 sequences, representing 6 countries), cluster E (13 sequences, representing 3 countries), cluster F (111 sequences, representing 4 countries), cluster G (143 sequences, representing 5 countries), cluster H (8 sequences, representing 4 countries), cluster I (17 sequences, representing 5 countries), cluster J (55 sequences, representing 4 countries), cluster K (130 sequences, representing 4 countries), cluster L (16 sequences, representing 2 countries), cluster M (84 sequences, representing 5 countries), cluster N (16 sequences, representing 3 countries), and cluster O (6 sequences, representing 1 countries). More than half of the sequences used to construct the tree were from Russia; therefore, the results show a predominance of Russian branches in most clusters (<a href="#fig3">Figure 3</a> and <a href="#tab3">Table 3</a>). Interestingly, sequences from Russia, Uzbekistan, and Latvia were intermixed in almost all the clusters, whereas sequences from Armenia were also represented prominently in most clusters (<a href="#fig3">Figure 3</a> and <a href="#tab3">Table 3</a>).</p> <p>The second tree was constructed with HIV <i>gag</i> sequences from 8 FSU countries spanning the years 1997â€"2013 as follows: Belarus (1997â€"2013), Georgia 1999 (1999), Kazakhstan (2002), Latvia (1998â€"2005), Russia (1997â€"2011), Ukraine (2000â€"2012), and Uzbekistan (2002). The majority of <i>gag</i> sequences were from Russia and Latvia, whereas only one sequence was from Georgia. Twelve clusters were picked from the tree for the further analysis (<a href="#tab4">Table 4</a> and <a href="#fig4">Figure 4</a>): cluster A (10 sequences, representing 1 countries), cluster B (13 sequences, representing 3 countries), cluster C (9 sequences, representing 4 countries), cluster D (7 sequences, representing 3 countries), cluster E (6 sequences, representing 4 countries), cluster F (17 sequences, representing 1 countries), cluster G (33 sequences, representing 4 countries), cluster H (13 sequences, representing 1 countries), cluster I (12 sequences, representing 2 countries), cluster J (20 sequences, representing 3 countries), cluster K (31 sequences, representing 1 countries), and cluster L (7 sequences, representing 3 countries). This tree clearly showed intermixing of Russian strains with Latvia (<a href="#fig4">Figure 4</a>). In agreement with the observations from the HIV <i>env</i> tree (<a href="#fig3">Figure 3</a>), in this tree as well, sequences from Russia and Latvia were intermixed in almost all the clusters throughout the tree (<a href="#fig4">Figure 4</a> and <a href="#tab4">Table 4</a>). Additionally, as was observed in the <i>env</i> tree, the ancestral sequences, representing the earliest infections, in this tree as well originated from Ukraine, with some intermixing of sequences from Russia (Figures <a href="#fig3">3</a> and <a href="#fig4">4</a>). Both in the <i>env</i> (cluster F) and <i>gag</i> (clusters A, F, H and K) trees, monophyletic clusters of Latvian sequences originating from Russian sequences were observed (Figures <a href="#fig3">3</a> and <a href="#fig4">4</a>), possibly indicating epidemics in Latvia that resulted from migrant-associated transmissions from Russia.</p> </div> <div class="3.3.riskgroupanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.3. Risk Group Analysis</div> <p>To analyse modes of transmission, high-risk data associated with HIV <i>env</i> and <i>gag</i> sequences were also analysed. Only sequences with recorded high-risk information were included in this analysis. Both the <i>env</i> (<a href="#fig5">Figure 5(a)</a>) and <i>gag</i> (<a href="#fig5">Figure 5(b)</a>) trees showed a predominance of PWID-associated sequences, followed by those linked with heterosexual transmission. Results represented here showed intermixing of PWID and heterosexual routes of transmission, indicating the possibility of bridging of the initial PWID-epidemic into heterosexual populations.</p> </div> <div class="3.4.bayesiananalysis" title="sec"> <div class="title" tagx="title" title="title">
3.4. Bayesian Analysis</div> <p>As summarized in <a href="#tab5">Table 5</a>, our Bayesian phylogenetic analysis showed substitution rate for NS5B HCV 1b and HCV 3a to be 6.8 (5.8â€"7.9)â€‰Ã—â€‰10<sup>âˆ’3</sup> and 1.1 (0.7â€"1.4)â€‰Ã—â€‰10<sup>âˆ’3</sup> substitutions per site per year, respectively, with a SkyGrid coalescent model, whereas the estimated tMRCAs for HCV 1b and HCV 3a were 1977 (95% highest posterior density, HPD: 1962â€"1985) and 1965 (95% HPD: 1941â€"1978), respectively. The estimated evolutionary rates of HIV <i>env</i> and HIV <i>gag</i> were 3.5 (2.5â€"4.0)â€‰Ã—â€‰10<sup>âˆ’2</sup> and 13 (8.8â€"17)â€‰Ã—â€‰10<sup>âˆ’3</sup> substitutions per site per year, respectively. The likely years of origins of HIV <i>env</i> and <i>gag</i> were estimated to be 1992 (95% HPD: 1990â€"1993) and 1996 (95% HPD: 1995â€"1997), respectively.</p> <p>We further investigated the past population dynamics of HCV and HIV using a Bayesian skyline plot, which depicts the changes in effective population size over time. The effective population size of HCV 1b was observed to be fluctuating and slowly rising from 1998 to 2011. The change in effective population size for HCV 3a rose slowly and steadily starting in 1965, characterized by a phase of exponential growth (1981â€"1997), and the effective population size surged approximately 100-fold during the period. The epidemic history of HIV <i>env</i> and HIV <i>gag</i> seems to have experienced complex dynamics, characterized by two phases of wave-like fluctuating growth, during 1997â€"2005 and 2009â€"2014 for HIV <i>env</i>, and during 1995â€"2005 and 2009â€"2015 for HIV <i>gag</i> (<a href="#fig6">Figure 6</a>).</p> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>In this study, we show evidence of transmission of two blood-borne viruses, HCV and HIV, within the FSU countries. According to our analysis, common routes of transmission appear to be injection drug use and heterosexual sex, with migrant mobility facilitating the cross-border spread of infection.</p> <p>There have been reports of high prevalence of blood-borne infections in the FSU region. Within FSU countries, Russia has the highest rate of HCV genotype 1b (50%), followed by HCV 3a (45%) [<a href="#B13">13</a>]. These two HCV genotypes also appear to be predominant in other FSU countries; Uzbekistan (1b, 64.2%; 3a, 25%), Azerbaijan (1b, 71.1%; 3a, 17%), Tajikistan (1b, 84.6%; 3a, 7.6%), Lithuania (1b, 54.3%; 3a, 23.9%), Estonia (1b, 71%; 3a, 24%), and Georgia (1b, 62%; 3a, 27%) [<a href="#B14">14</a>â€"<a href="#B19">19</a>].</p> <p>In our phylogenetic tree constructed with HCV 1b sequences (<a href="#fig1">Figure 1</a>), a preponderance of Russian sequences was seenâ€"a reflection of a higher representation of sequences from this country in the Los Alamos database. Russian sequences were intermixed with clusters from most other FSU countries, with a relatively higher proportion from Lithuania and Estonia. This might reflect a higher transmigration between Russia, Lithuania, and Estonia, due to their close geographic proximity. Conversely, in clusters C, D, and F, sequences from Azerbaijan, Uzbekistan, and Tajikistan were seen intermixed with those from Russia, indicating that Russia might have served as a hub for transmission across these countries. Phylogenetic tree constructed with HCV 3a NS5B sequences (<a href="#fig2">Figure 2</a>) showed an overall agreement with the 1b tree, showing trends of possible transmission within Russia, Lithuania, Uzbekistan, Estonia, and Azerbaijan.</p> <p>Such phylogenetic patterns may be supported by historical and socioeconomic factors that came into play after the disintegration of the Soviet Union. It has been established that the first nationalism-related conflicts between Armenia and Azerbaijan (1987â€"1991), followed by Uzbekistan conflict (1989), and civil wars in Georgia (1991â€"1993) and Tajikistan (1992â€"1997) led to mass migration from these countries to Russia [<a href="#B1">1</a>]. Sequences from these countries that show intermixing with those from Russia also date to roughly the same time (Tables <a href="#tab1">1</a> and <a href="#tab2">2</a>). Our Bayesian analysis predicts origin of the HCV epidemics in FSU countries to be as early as 1965, for HCV3a, and 1977, for genotype 1b (<a href="#tab5">Table 5</a> and <a href="#fig6">Figure 6</a>), indicating that these HCV genotypes were possibly circulating in the region before the conflicts started. It is possible that HCV was transmitted during this time through injection use, leading to a rise in transmission in the mid-1990s, as suggested by our Skyline plot analysis (<a href="#fig6">Figure 6</a>). In addition, significant reduction of employment opportunities and a dramatic decline in wages in Tajikistan and Georgia also facilitated a high flow of migrant labor into Russia. Further movement involved repatriates and internally displaced people of Russian-speaking origin into Russia owing to a resurgence of nationalism as a result of implementation of national language policies that discouraged use of Russian. It is thought that the reported numbers of migrants in this region is much lower than the actual numbers, since visa-free movement across borders does not account for illegally entering migrant workers [<a href="#B1">1</a>].</p> <p>In the initial stages of HIV outbreak in central Asian region, most people infected with HIV acquired a homogenic variant of subtype A, known as A<sub>FSU</sub> (A6). This was thought to be a large-scale founder effect, resulting from transmigration within FSU countries [<a href="#B20">20</a>]. The first HIV epidemic in FSU was recorded in the mid-1990s in Ukraine. A year later, outbreaks were detected in the European part of Russia, thereafter reaching other parts of Russia, Central Asia, and Eastern Europe.</p> <p>The HIV epidemic expanded rapidly after the collapse of the USSR. As noted previously, political transitions led to the socioeconomic decline of newly formed independent counties. Consequently, large-scale labor migration to Russia from other countries of FSU provided epidemiological bridging of HIV transmission between Russia and the rest of FSU countries [<a href="#B20">20</a>]. The database of international migration demonstrated a significant increase in migration between 1991 and 2000 within the FSU. The number of migrants from Central Asia, Caucasus, and Eastern Europe to Russia was highest from 1992 to 1996 [<a href="#B21">21</a>], the same period during which our Skyline plot shows a gradual rise in the infections (<a href="#fig6">Figure 6</a>). Additionally, our phylogenetic trees revealed a dense intermixing of Russian sequences with other countries, dating around the same time (Figures <a href="#fig3">3</a> and <a href="#fig4">4</a> and Tables <a href="#tab3">3</a> and <a href="#tab4">4</a>), which might suggest a link to migrant-associated HIV transmission. The HIV epidemic in FSU is considered to have initiated by transmissions through sexual contact and injection drug use [<a href="#B20">20</a>]. Our results of phylogenetic tree in the context of high-risk behavior (Figures <a href="#fig5">5(a)</a> and <a href="#fig5">5(b)</a>) support that fact, showing the branches close to the ancestral node associated with heterosexual mode of transmission, with sequences represented from Uzbekistan (1999), Ukraine (1993), and Russia (1994).</p> <p>As shown in previous reports, the initial HIV outbreak in FSU countries occurred in the mid-1990s in the city of Odessa, Ukraine, and spread to other countries of FSU [<a href="#B22">22</a>]. Our phylogenetic tree also shows the ancestral node for the HIV subtype A epidemic originating from Ukrainian sequences deposited in 1993 (Figures <a href="#fig3">3</a> and <a href="#fig4">4</a>), whereas the Bayesian analysis also predicts the origins of the epidemic to be around the same time (<a href="#tab1">Table 1</a> and <a href="#fig6">Figure 6</a>). These results might indicate that Odessa had a favourable location for an explosive spread of HIV among PWID, as the city is situated on the Black Sea shore and was a major seaport and transportation hub during that period [<a href="#B20">20</a>]. In the first postcollapse decade, there was an increase in injection drug use in the FSU [<a href="#B23">23</a>], linked to the rise of drug production in Afghanistan during the mid-1990s and a shift in trafficking to the â€œNorthern route,â€� which runs through Central Asia to Russia and South Caucasus to Eastern Europe [<a href="#B24">24</a>, <a href="#B25">25</a>]. This increase in drug trafficking may have played a role in the early spread of HIV among PWID communities. Our phylogenetic tree also shows a high number of transmissions associated with drug use, later shifting the mode of transmission toward heterosexual sex. Furthermore, our phylogenetic analysis shows heavy intermixing of Armenian PWID sequences from the South Caucasian region, which is within â€œNorthern routeâ€� of drug trafficking, with other Eastern European sequences. During the period of 1989â€"2001, another important scenario that may have facilitated transmission of HIV and HCV was the displacement of a population of about one million Armenians in aftermath of the 1988 earthquake, coupled with deteriorating socioeconomic situation in the 1990s. These displaced labor populations immigrated mainly to the Eastern European countries of Ukraine, Belarus, and Eastern part of Russia [<a href="#B26">26</a>].</p> </div> <div class="5.conclusion" title="sec"> <div class="title" tagx="title" title="title">
5. Conclusion</div> <p>Our analyses highlight the main routes of transmission for two important blood-borne viruses in FSU countries. This information may be exploited to refine Public Health policies to better manage the infected populations and prevent further spread of the infections. Although this study was limited by the availability of the number of sequences from FSU countries in the open-access databases, the results highlight the importance of focusing the harm reduction efforts on communities where blood-borne infections might be currently spreading, namely, persons who inject drugs, heterosexual populations, migrant workers, and cross-border travelers.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>Funding for this study was partially provided by Nazarbayev University under grant: 110119FD4516. SHV is supported, in part, by NIH Grant No. P30MH062294. JD was partially supported by Fogarty International Center and the National Institute on Drug Abuse of the National Institutes of Health under Award Number D43 TW010046.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The data used to support the findings of this study are available from the corresponding author.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that they have no conflicts of interest.</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' Contributions</div> <p>Lazzat Aibekova, Aizada Bexeitova, and Arailym Aldabergenova have contributed equally to the manuscript.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Materials</span> <p>Supplementary Figure 1: numbers of HIV and HCV sequences available in Los Alamos Database are shown for each country. HCV Genotype 1b NS5B, HCV Genotype 3a NS5B, HIV Subtype A env, and HIV Subtype A gag are color-coded in boxes as yellow, green, light blue, and dark blue, respectively.</p>  <div class="media" title="media"><a href="9701920.f1.pptx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudinovskikh</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denissenko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="source" tagx="source" title="source"><i>Migration between CIS Countries: Trends and Policy</i></span><span class="year" tagx="year" title="year">2014</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Moscow, Russia</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">National Research University</span></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laga</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilyev</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapovok</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HIV type 1 subtype A1 dominates in Armenia</span><span class="source" tagx="source" title="source"><i>Current HIV Research</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">219</span><span class="lpage" tagx="lpage" title="lpage">225</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/1570162x13666150407142834">10.2174/1570162x13666150407142834</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25845390">25845390</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manuylov</span><span class="given-names" tagx="given-names" title="given-names">V. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chub</span><span class="given-names" tagx="given-names" title="given-names">E. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kichatova</span><span class="given-names" tagx="given-names" title="given-names">V. S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus subtype 3a was introduced in the USSR in the early 1980s</span><span class="source" tagx="source" title="source"><i>Journal of General Virology</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">98</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">2079</span><span class="lpage" tagx="lpage" title="lpage">2087</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000878">10.1099/jgv.0.000878</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28742003">28742003</a></span></span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renton</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gzirishvilli</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotsadze</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godinho</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemics of HIV and sexually transmitted infections in Central Asia: trends, drivers and priorities for control</span><span class="source" tagx="source" title="source"><i>International Journal of Drug Policy</i></span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">494</span><span class="lpage" tagx="lpage" title="lpage">503</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.drugpo.2006.09.003">10.1016/j.drugpo.2006.09.003</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source"><i>Guidelines for the Screening, Care and Treatment of Persons with Chronic Hepatitis C Infection</i></span><span class="year" tagx="year" title="year">2016</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva, Switzerland</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span></span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="collab'">collab: UNAIDS</span><span class="source" tagx="source" title="source"><i>UNAIDS Data 2017</i></span><span class="year" tagx="year" title="year">2017</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva, Switzerland</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">UNAIDS</span><span class="comment"><a href="http://www.unaids.org/sites/default/files/media_asset/20170720_Data_book_2017_en.pdf">http://www.unaids.org/sites/default/files/media_asset/20170720_Data_book_2017_en.pdf</a></span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tracing the epidemic history of HIV-1 CRF01_AE clusters using near-complete genome sequences</span><span class="source" tagx="source" title="source"><i>Scientific Reports</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">p. 4024</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-03820-8">10.1038/s41598-017-03820-8</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RodrÃ­guez</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliver</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MarÃ­n</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medina</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The general stochastic model of nucleotide substitution</span><span class="source" tagx="source" title="source"><i>Journal of Theoretical Biology</i></span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">142</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">485</span><span class="lpage" tagx="lpage" title="lpage">501</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0022-5193(05)80104-3">10.1016/s0022-5193(05)80104-3</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2338834">2338834</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods</span><span class="source" tagx="source" title="source"><i>Journal of Molecular Evolution</i></span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">39</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">306</span><span class="lpage" tagx="lpage" title="lpage">314</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/bf00160154">10.1007/bf00160154</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7932792">7932792</a></span></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">S. Y. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Relaxed phylogenetics and dating with confidence</span><span class="source" tagx="source" title="source"><i>PLoS Biology</i></span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">p. e88</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.0040088">10.1371/journal.pbio.0040088</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O. G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences</span><span class="source" tagx="source" title="source"><i>Molecular Biology and Evolution</i></span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1185</span><span class="lpage" tagx="lpage" title="lpage">1192</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msi103">10.1093/molbev/msi103</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703244">15703244</a></span></span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">N. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The early spread and epidemic ignition of HIV-1 in human populations</span><span class="source" tagx="source" title="source"><i>Science</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">346</span><span class="issue" tagx="issue" title="issue">6205</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1256739">10.1126/science.1256739</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25278604">25278604</a></span></span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalinina</span><span class="given-names" tagx="given-names" title="given-names">O. V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus: variability mechanisms, classification, evolution</span><span class="source" tagx="source" title="source"><i>Voprosy Virusologii</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">60</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">10</span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurbanov</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugauchi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus molecular epidemiology in Uzbekistan</span><span class="source" tagx="source" title="source"><i>Journal of Medical Virology</i></span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">69</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">367</span><span class="lpage" tagx="lpage" title="lpage">375</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.10298">10.1002/jmv.10298</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12526047">12526047</a></span></span></li> <li tag="ref"><a name="B15" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bokharaei-Salim</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keyvani</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monavari</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alavian</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fakhim</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nasseri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distribution of hepatitis C virus genotypes among Azerbaijani patients in capital city of Iran-tehran</span><span class="source" tagx="source" title="source"><i>Hepatitis Monthly</i></span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">9</span><span class="pub-id" /><span class="pub-id"><a href="https://dx.doi.org/10.5812/hepatmon.13699">10.5812/hepatmon.13699</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B16" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurbanov</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiological and clinical evaluation of hepatitis B, hepatitis C, and delta hepatitis viruses in Tajikistan</span><span class="source" tagx="source" title="source"><i>Journal of Medical Virology</i></span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">80</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">268</span><span class="lpage" tagx="lpage" title="lpage">276</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.21057">10.1002/jmv.21057</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18098133">18098133</a></span></span></li> <li tag="ref"><a name="B17" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ambrozaitis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ZÌ†agminas</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BalcÌ†iÅ«naite</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widell</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C in Lithuania: incidence, prevalence, risk factors and viral genotypes</span><span class="source" tagx="source" title="source"><i>Clinical and Diagnostic Virology</i></span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">273</span><span class="lpage" tagx="lpage" title="lpage">284</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0928-0197(95)00017-8">10.1016/0928-0197(95)00017-8</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15566848">15566848</a></span></span></li> <li tag="ref"><a name="B18" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tallo</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Norder</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tefanova</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of hepatitis C virus strains in Estonia: fluctuations in the predominating subtype with time</span><span class="source" tagx="source" title="source"><i>Journal of Medical Virology</i></span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">79</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">374</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.20828">10.1002/jmv.20828</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17311333">17311333</a></span></span></li> <li tag="ref"><a name="B19" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petruzziello</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marigliano</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loquercio</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cozzolino</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cacciapuoti</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global epidemiology of hepatitis C virus infection: an up-date of the distribution and circulation of hepatitis C virus genotypes</span><span class="source" tagx="source" title="source"><i>World Journal of Gastroenterology</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">34</span><span class="fpage" tagx="fpage" title="fpage">7824</span><span class="lpage" tagx="lpage" title="lpage">7840</span><span class="pub-id"><a href="https://dx.doi.org/10.3748/wjg.v22.i34.7824">10.3748/wjg.v22.i34.7824</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27678366">27678366</a></span></span></li> <li tag="ref"><a name="B20" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bobkova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Current status of HIV-1 diversity and drug resistance monitoring in the former USSR</span><span class="source" tagx="source" title="source"><i>AIDS Reviews</i></span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">204</span><span class="lpage" tagx="lpage" title="lpage">212</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24192601">24192601</a></span></span></li> <li tag="ref"><a name="B21" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="comment">UN International Migration Flows to and from Selected Countries: The 2015 Revision, 2015</span></span></li> <li tag="ref"><a name="B22" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DÃ­ez-Fuertes</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabello</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomson</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian phylogeographic analyses clarify the origin of the HIV-1 subtype A variant circulating in former Soviet Unionâ€™s countries</span><span class="source" tagx="source" title="source"><i>Infection, Genetics and Evolution</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2015.05.003">10.1016/j.meegid.2015.05.003</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B23" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhodes</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ball</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stimson</span><span class="given-names" tagx="given-names" title="given-names">G. V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HIV infection associated with drug injecting in the newly independent states, eastern Europe: the social and economic context of epidemics</span><span class="source" tagx="source" title="source"><i>Addiction</i></span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">94</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">1323</span><span class="lpage" tagx="lpage" title="lpage">1336</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1360-0443.1999.94913235.x">10.1046/j.1360-0443.1999.94913235.x</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10615718">10615718</a></span></span></li> <li tag="ref"><a name="B24" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="collab'">collab: UNODC</span><span class="source" tagx="source" title="source"><i>Global Illicit Drug Trends: 2003</i></span><span class="year" tagx="year" title="year">2003</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">UNODC</span></span></li> <li tag="ref"><a name="B25" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farooq</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasooly</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abidi</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modjarrad</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Opium trade and the spread of HIV in the golden crescent</span><span class="source" tagx="source" title="source"><i>Harm Reduction Journal</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">p. 47</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12954-017-0170-1">10.1186/s12954-017-0170-1</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B26" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="collab'">collab: International Labour Organization</span><span class="source" tagx="source" title="source"><i>ILO Migration and Development: Armenia Country Study</i></span><span class="year" tagx="year" title="year">2009</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Yerevan, Armenia</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">International Labour Organization</span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Phylogenetic relationship of HCV genotype 1b NS5B sequences from FSU countries: For the construction of the phylogenetic tree, a stretch of HCV sequence corresponding to H77 nucleotide 8322â€"8555 was used. The sequences are included from Azerbaijan, Georgia, Estonia, Lithuania, Russia, Tajikistan, and Uzbekistan, represented by, respectively, orange, purple, pink, brown, dark blue, coral, and turquoise branches. The branches coded grey represent 10 outgroup reference sequences that were used to root the tree.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Phylogenetic relationship of HCV genotype 3a NS5B sequences from FSU countries: For the construction of the phylogenetic tree, a stretch of HCV sequence corresponding to H77 nucleotide 8322â€"8555 was used. The sequences are included from Azerbaijan, Georgia, Estonia, Lithuania, Russia, Tajikistan, and Uzbekistan. The color key is the same as for <a href="#fig1">Figure 1</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Phylogenetic relationship of HIV subtype A <i>env</i> gene sequences from FSU countries: For the construction of the phylogenetic tress, a stretch of HIV sequence corresponding to HBX2 nucleotide 7071â€"7336 was used. The sequences are included from Armenia, Belarus, Georgia, Estonia, Kyrgyzstan, Latvia, Moldova, Russia, Ukraine and Uzbekistan represented by, respectively, teal, green, purple, pink, neon, yellow, olive, dark blue, red, and turquoise branches. The branches coded grey represent 12 outgroup reference sequences that were used to root the tree.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Phylogenetic relationship of HIV subtype A <i>gag</i> gene sequences from FSU countries: For the construction of the phylogenetic tress, a stretch of HIV sequence corresponding to HBX2 nucleotide 859â€"1230 was used. The sequences are included from Belarus, Georgia, Kazakhstan, Latvia, Russia, Ukraine, and Uzbekistan. Color key is the same as for <a href="#fig3">Figure 3</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Analysis of HIV transmission routes in FSU countries: HIV-1 subtype A <i>env</i> (a) or <i>gag</i> (b) sequences from FSU countries were analysed for high-risk behavior. To analyse risk behavior in the context of location, the two versions of the same tree, with branches color-coded for the country of origin (left) or the associated risk group (right), are juxtaposed. Only sequences with recorded high-risk labels were included in this analysis. Risk behavior information for each sequence was obtained from HIV Los Alamos Database. For the tree on the right, the colors dark blue, teal, turquoise, pink, green, olive, red, neon, orange, and grey branches indicate, respectively, heterosexual, sexual transmission (unspecified type), MSM, mother-to-child, PWID, blood transfusion, homosexual, sex worker, bisexual, and reference sequences. For the tree on the left, the color key is the same as for <a href="#fig4">Figure 4</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Population dynamics of the expansion of HCV genotypes 1B and 3A (top left and right panel, respectively) (NS5B) and HIV-1 subtype A (<i>env</i> and <i>gag</i>: bottom left and right panel, respectively). The population dynamics was analysed by Bayesian MCMC inference method with a SkyGrid demographic reconstruction model in BEAST. The vertical axis shows the effective number of infections (<i>N</i><sub><i>e</i></sub>) multiplied by mean viral generation time (Ñ‚). The solid line and shaded region represent the median and 95% credibility interval of <i>N</i><sub><i>e</i></sub>Ñ‚ through time.</p>   </div> <div class="table-wrap_UNKNOWN" id="tab1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Cluster analysis of HCV 1b <i>NS5B</i> tree in <a href="#fig1">Figure 1</a> showing distribution of sequences among FSU countries.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Cluster</th> <th align="center" rowspan="1" colspan="1">Number of sequences</th> <th align="center" rowspan="1" colspan="1">Number of countries</th> <th align="center" rowspan="1" colspan="1">Countries</th> <th align="center" rowspan="1" colspan="1">Years</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="center" rowspan="1" colspan="1">53</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Tajikistan</td> <td align="center" rowspan="1" colspan="1">1997â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="center" rowspan="1" colspan="1">37</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Tajikistan, Georgia, Lithuania</td> <td align="center" rowspan="1" colspan="1">1997â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="center" rowspan="1" colspan="1">18</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Azerbaijan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1999â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="center" rowspan="1" colspan="1">23</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1997â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="center" rowspan="1" colspan="1">133</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1995â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="center" rowspan="1" colspan="1">33</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1997â€"2006</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="center" rowspan="1" colspan="1">30</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1998â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="center" rowspan="1" colspan="1">31</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Estonia, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1998â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">I</td> <td align="center" rowspan="1" colspan="1">39</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Azerbaijan, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1995â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="center" rowspan="1" colspan="1">32</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Tajikistan, Lithuania</td> <td align="center" rowspan="1" colspan="1">1997â€"2014</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Cluster analysis of HCV 3a <i>NS5B</i> tree in <a href="#fig2">Figure 2</a> showing distribution of sequences among FSU countries.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Cluster</th> <th align="center" rowspan="1" colspan="1">Number of sequences</th> <th align="center" rowspan="1" colspan="1">Number of countries</th> <th align="center" rowspan="1" colspan="1">Countries</th> <th align="center" rowspan="1" colspan="1">Years</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="center" rowspan="1" colspan="1">44</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Lithuania, Azerbaijan, Georgia</td> <td align="center" rowspan="1" colspan="1">1998â€"2012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Lithuania, Russia, Azerbaijan</td> <td align="center" rowspan="1" colspan="1">1998â€"2011</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="center" rowspan="1" colspan="1">11</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Lithuania, Estonia</td> <td align="center" rowspan="1" colspan="1">2001â€"2006</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="center" rowspan="1" colspan="1">18</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Estonia, Azerbaijan</td> <td align="center" rowspan="1" colspan="1">1998â€"2009</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="center" rowspan="1" colspan="1">23</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia</td> <td align="center" rowspan="1" colspan="1">1998â€"2009</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="center" rowspan="1" colspan="1">80</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Estonia, Lithuania, Azerbaijan, Tajikistan</td> <td align="center" rowspan="1" colspan="1">1999â€"2014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="center" rowspan="1" colspan="1">33</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Uzbekistan, Russia, Azerbaijan, Estonia</td> <td align="center" rowspan="1" colspan="1">1999â€"2008</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3</span> <p>Cluster analysis of HIV <i>env</i> tree in <a href="#fig3">Figure 3</a> showing distribution of sequences among FSU countries.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Cluster</th> <th align="center" rowspan="1" colspan="1">Number of sequences</th> <th align="center" rowspan="1" colspan="1">Number of countries</th> <th align="center" rowspan="1" colspan="1">Countries</th> <th align="center" rowspan="1" colspan="1">Years</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="center" rowspan="1" colspan="1">76</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Russia, Uzbekistan, Estonia, Kyrgyzstan, Armenia</td> <td align="center" rowspan="1" colspan="1">1999â€"2012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="center" rowspan="1" colspan="1">53</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Uzbekistan, Estonia</td> <td align="center" rowspan="1" colspan="1">1999â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="center" rowspan="1" colspan="1">25</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia</td> <td align="center" rowspan="1" colspan="1">2001â€"2012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="center" rowspan="1" colspan="1">83</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Uzbekistan, Belarus, Ukraine, Moldova</td> <td align="center" rowspan="1" colspan="1">1996â€"2012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Uzbekistan, Ukraine</td> <td align="center" rowspan="1" colspan="1">1995â€"2012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="center" rowspan="1" colspan="1">111</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Armenia, Estonia</td> <td align="center" rowspan="1" colspan="1">1998â€"2013</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="center" rowspan="1" colspan="1">143</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Uzbekistan, Armenia, Ukraine</td> <td align="center" rowspan="1" colspan="1">1993â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Uzbekistan, Armenia</td> <td align="center" rowspan="1" colspan="1">1999â€"2009</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">I</td> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Uzbekistan, Moldova, Armenia</td> <td align="center" rowspan="1" colspan="1">1997â€"2011</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="center" rowspan="1" colspan="1">55</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Armenia, Belarus</td> <td align="center" rowspan="1" colspan="1">2004â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">K</td> <td align="center" rowspan="1" colspan="1">130</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Armenia, Ukraine</td> <td align="center" rowspan="1" colspan="1">2001â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">L</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia</td> <td align="center" rowspan="1" colspan="1">2001â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">M</td> <td align="center" rowspan="1" colspan="1">84</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Armenia, Ukraine, Georgia</td> <td align="center" rowspan="1" colspan="1">1999â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">N</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Armenia, Ukraine</td> <td align="center" rowspan="1" colspan="1">2009â€"2015</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">O</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">Ukraine</td> <td align="center" rowspan="1" colspan="1">1993</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab4" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4</span> <p>Cluster analysis of HIV <i>gag</i> tree in <a href="#fig4">Figure 4</a> showing distribution of sequences among FSU countries.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Cluster</th> <th align="center" rowspan="1" colspan="1">Number of sequences</th> <th align="center" rowspan="1" colspan="1">Number of countries</th> <th align="center" rowspan="1" colspan="1">Countries</th> <th align="center" rowspan="1" colspan="1">Years</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">Latvia</td> <td align="center" rowspan="1" colspan="1">2001â€"2005</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Ukraine</td> <td align="center" rowspan="1" colspan="1">2000â€"2003</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Kazakhstan, Uzbekistan</td> <td align="center" rowspan="1" colspan="1">2002â€"2005</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Ukraine</td> <td align="center" rowspan="1" colspan="1">2001â€"2011</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Ukraine, Belarus</td> <td align="center" rowspan="1" colspan="1">1997â€"2005</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">Latvia</td> <td align="center" rowspan="1" colspan="1">1998â€"2002</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="center" rowspan="1" colspan="1">33</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Ukraine, Belarus</td> <td align="center" rowspan="1" colspan="1">1997â€"2006</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">Latvia</td> <td align="center" rowspan="1" colspan="1">2001â€"2003</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">I</td> <td align="center" rowspan="1" colspan="1">12</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">Latvia, Russia</td> <td align="center" rowspan="1" colspan="1">2001â€"2005</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="center" rowspan="1" colspan="1">20</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Latvia, Ukraine</td> <td align="center" rowspan="1" colspan="1">1998â€"2008</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">K</td> <td align="center" rowspan="1" colspan="1">31</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">Latvia</td> <td align="center" rowspan="1" colspan="1">2001â€"2005</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">L</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">3</td> <td align="center" rowspan="1" colspan="1">Russia, Kazakhstan, Latvia</td> <td align="center" rowspan="1" colspan="1">2002â€"2006</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab5" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5</span> <p>tMRCA and Substitution rate of HCV and HIV.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">â€‰</th> <th align="center" rowspan="1" colspan="1">tMRCA (median and 95% HPD interval)</th> <th align="center" rowspan="1" colspan="1">Substitution/site/year (median and 95% HPD interval)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">HCV 1B</td> <td align="center" rowspan="1" colspan="1">37.7 (30.0, 52.6)</td> <td align="center" rowspan="1" colspan="1">6.8<i>E</i>â€‰â€"â€‰3 (5.8<i>E</i>â€‰â€"3, 7.9<i>E</i>â€‰â€"â€‰3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HCV 3A</td> <td align="center" rowspan="1" colspan="1">49.3 (36.0, 73.1)</td> <td align="center" rowspan="1" colspan="1">1.1<i>E</i>â€‰â€"â€‰3 (7.0<i>E</i>â€‰â€"â€‰4, 1.4<i>E</i>â€‰â€"â€‰3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HIV <i>ENV</i></td> <td align="center" rowspan="1" colspan="1">22.8 (22.0, 24.8)</td> <td align="center" rowspan="1" colspan="1">3.5<i>E</i>â€‰â€"â€‰2 (2.5<i>E</i>â€‰â€"â€‰2, 4.0<i>E</i>â€‰â€"â€‰2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HIV <i>GAG</i></td> <td align="center" rowspan="1" colspan="1">16.9 (16.3, 17.5)</td> <td align="center" rowspan="1" colspan="1">1.3<i>E</i>â€‰â€"â€‰2 (8.8<i>E</i>â€‰â€"â€‰3, 1.7<i>E</i>â€‰â€"â€‰2)</td> </tr> </tbody> </table> </div> </div>  </body></html>